webapp/apps/btax/compute.py

```diff
@@ -0,0 +1,57 @@
+import dropq
+import os
+from ..taxbrain.models import WorkerNodesCounter
+import json
+import requests
+from requests.exceptions import Timeout, RequestException
+from .helpers import arrange_totals_by_row
+from ..taxbrain.compute import (DropqCompute,
+                                MockCompute,
+                                MockFailedCompute,
+                                NodeDownCompute,
+                                JobFailError,
+                                ENFORCE_REMOTE_VERSION_CHECK,
+                                TIMEOUT_IN_SECONDS,
+                                dropq_version)
+import requests_mock
+requests_mock.Mocker.TEST_PREFIX = 'dropq'
+
+
+
+def package_up_vars(self, user_mods, first_budget_year):
```

Comment:
My status is that I got the Travis CI tests to run okay, I can run the webapp/deploy/btax/ogusa system locally for informal testing, but I still cannot get pytest to run the tests locally okay.  What I try to do locally is run the commands from the bottom of the .travis.yml.  Here's what I'm trying and the errors I'm getting:
```
python manage.py syncdb --noinput && python manage.py makemigrations taxbrain && python manage.py migrate taxbrain && python manage.py migrate --fake &&  py.test webapp/apps/
/Users/psteinberg/miniconda3/envs/webapp/lib/python2.7/site-packages/django/core/management/commands/syncdb.py:24: RemovedInDjango19Warning: The syncdb command will be removed in Django 1.9
  warnings.warn("The syncdb command will be removed in Django 1.9", RemovedInDjango19Warning)

Operations to perform:
  Synchronize unmigrated apps: gunicorn, staticfiles, messages, humanize, import_export, btax, storages
  Apply all migrations: account, sessions, admin, flatblocks, register, dynamic, sites, auth, taxbrain, contenttypes, hermes
Synchronizing apps without migrations:
  Creating tables...
    Running deferred SQL...
  Installing custom SQL...
Running migrations:
  No migrations to apply.
No changes detected in app 'taxbrain'
Operations to perform:
  Apply all migrations: taxbrain
Running migrations:
  No migrations to apply.
Operations to perform:
  Synchronize unmigrated apps: gunicorn, staticfiles, messages, humanize, import_export, btax, storages
  Apply all migrations: account, sessions, admin, flatblocks, register, dynamic, sites, auth, taxbrain, contenttypes, hermes
Synchronizing apps without migrations:
  Creating tables...
    Running deferred SQL...
  Installing custom SQL...
Running migrations:
  No migrations to apply.
============================= test session starts ==============================
platform darwin -- Python 2.7.12, pytest-2.9.2, py-1.4.31, pluggy-0.3.1
rootdir: /Users/psteinberg/Documents/AEI/webapp-public, inifile: pytest.ini
plugins: xdist-1.15.0
collected 1 items / 7 errors

webapp/apps/dynamic/tests/test_all.py E

==================================== ERRORS ====================================
____________ ERROR collecting webapp/apps/btax/tests/test_views.py _____________
webapp/apps/btax/tests/test_views.py:5: in <module>
    from ..models import BTaxSaveInputs, BTaxOutputUrl
webapp/apps/btax/models.py:8: in <module>
    from django.contrib.auth.models import User
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/contrib/auth/__init__.py:7: in <module>
    from django.middleware.csrf import rotate_token
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/middleware/csrf.py:14: in <module>
    from django.utils.cache import patch_vary_headers
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/utils/cache.py:26: in <module>
    from django.core.cache import caches
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/core/cache/__init__.py:34: in <module>
    if DEFAULT_CACHE_ALIAS not in settings.CACHES:
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/conf/__init__.py:48: in __getattr__
    self._setup(name)
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/conf/__init__.py:42: in _setup
    % (desc, ENVIRONMENT_VARIABLE))
E   ImproperlyConfigured: Requested setting CACHES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.
________ ERROR collecting webapp/apps/dynamic/tests/test_behavioral.py _________
webapp/apps/dynamic/tests/test_behavioral.py:9: in <module>
    from ...taxbrain.models import TaxSaveInputs
webapp/apps/taxbrain/models.py:8: in <module>
    from django.contrib.auth.models import User
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/contrib/auth/__init__.py:7: in <module>
    from django.middleware.csrf import rotate_token
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/middleware/csrf.py:14: in <module>
    from django.utils.cache import patch_vary_headers
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/utils/cache.py:26: in <module>
    from django.core.cache import caches
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/core/cache/__init__.py:34: in <module>
    if DEFAULT_CACHE_ALIAS not in settings.CACHES:
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/conf/__init__.py:48: in __getattr__
    self._setup(name)
../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/conf/__init__.py:42: in _setup
    % (desc, ENVIRONMENT_VARIABLE))
E   ImproperlyConfigured: Requested setting CACHES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.

....more such errors

>               % (desc, ENVIRONMENT_VARIABLE))
E           ImproperlyConfigured: Requested setting DATABASES, but settings are not configured. You must either define the environment variable DJANGO_SETTINGS_MODULE or call settings.configure() before accessing settings.

../../../miniconda3/envs/webapp/lib/python2.7/site-packages/django/conf/__init__.py:42: ImproperlyConfigured
=========================== 8 error in 1.62 seconds ============================
(webapp)
```

I've been looking for help on this Django settings.py module error and everything says exactly what the error messages above tell me to do (call `configure` or define the env var `DJANGO_SETTINGS_MODULE`).  We have pytest.ini specifying the path to settings and I get the same error with the pytest.ini or when I move it out of the way.  Theoretically our pytest.ini is telling pytest where our settings.py is located and DJANGO_SETTINGS_MODULE env var is unneccesary.